packages/iris-grid/src/sidebar/TableSaver.tsx

```diff
@@ -142,29 +142,29 @@ export default class TableSaver extends PureComponent<
     if (this.snapshotHandlerTimeout) clearTimeout(this.snapshotHandlerTimeout);
   }
 
-  sw: ServiceWorker | null;
+  sw: ServiceWorker | undefined;
```

Comment:
Prefer to define as optional instead of `| undefined`
```suggestion
  sw?: ServiceWorker;
```
Same with all the others below